dts/bindings/sensor/vishay,veml7700.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2022 Nikolaus Huber
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Driver for Vishay VEML7700 High-Accuracy Ambient Light Sensor. See datasheet
+  at https://www.vishay.com/docs/84286/veml7700.pdf
+
+compatible: "vishay,veml7700"
+
+include: i2c-device.yaml
+
+properties:
+  als-it:
+    type: int
+    required: false
+    default: 100
```

Comment:
Please document why this value was selected as the default.
https://docs.zephyrproject.org/latest/build/dts/bindings.html#default-values-for-properties